### ISSUE TITLE
Showing PMD results in the code as coloured lines?

### ISSUE THREAD

**Issue by a-t-0** [amitdev/PMD-Intellij#177]
Hi, thank you for your work on embedding PMD with IntelliJ! 

Is it possible to show the output of PMD (live) in the IDE code with coloured lines? Similar to how other code-issues are visualised:

![image](https://github.com/user-attachments/assets/f93aebba-8405-413c-b7b1-73f523beabdb)

I am aware it may take a while to update each file, luckily PMD supports incremental analysis, and perhaps one could set an update frequency, e.g. once every 2 minutes for changed files (w.r.t. the previous run).  

If this is already possible, would you perhaps be willing to explain how to configure this setup? 

(For completeness, I am able to manually run PMD in IntelliJ, and inspect the issues, and hop to the lines that are affected.)

**Comment by martenbohlin** [comment]
Yes it is possible today. In the settings go to Tools -> PMD -> Options. Use ctrl (or cmd on mac) and click on the rule-sets you want to be highlighted in the editor. In the example below bestpractices,  codestyle, and design warnings will be shown directly in the editor

<img width="981" alt="image" src="https://github.com/user-attachments/assets/dc3015f9-850e-4de7-b896-f0a31ab71c9e" />

**Comment by a-t-0** [comment]
Thank you Marten, for sharing such a detailed explanation!
